eea/facetednavigation/widgets/storage.py

```diff
@@ -17,6 +17,7 @@ class Criterion(object):
     position = u'right'
     catalog = None
     _hidden = False
+    custom_css = u''
```

Comment:
Ha, now I get it. You added to the Storage class...

Reply:
Yes... I was struggling with the NoneType before, but this should solve that.